docs/source/interfaces.rst

```diff
@@ -0,0 +1,12 @@
+Interfaces
+==========
+
+Getting the interfaces provided by the current context:
```

Comment:
@erral we usually try to add a short introduction to non-Plone devs what this documentation section is about. E.g. a short introduction what interfaces are, maybe with a link to the Plone docs.